ESP32LapTimer/src/ESP32LapTimer.cpp

```diff
@@ -74,7 +69,22 @@ void setup() {
   //PowerDownAll(); // Powers down all RX5808's
   delay(250);
 
+#if defined(WIFI_MODE_ACESSPOINT)
   InitWifiAP();
+#elif defined(WIFI_MODE_CLIENT)
+  if(!InitWifiClient()) {
+    Serial.println("Failed to connect to WiFi Network");
+    Serial.println("Starting up in AP mode instead!");
+    InitWifiAP();
+  } else {
+    WiFi.onEvent([](WiFiEvent_t event, WiFiEventInfo_t info) {
+      log_i("WiFi network disconnected, retrying...");
+      WiFi.begin(WIFI_SSID, WIFI_PASSWORD);
+    }, WiFiEvent_t::SYSTEM_EVENT_STA_DISCONNECTED);
+  }
+#else
+#error "No WIFI_MODE selected"
```

Comment:
Thoughts about doing this in the target.h file? We could use this as a place to verify if the config is valid

Reply:
Probably a good idea but won't that cause some kind of circular dependency as it currently stands? target.h needs to import hardwareconfig.h which imports target.h and so on.

Reply:
Hardwareconfig.h imports target.h, which imports the device specific configs and defines some stuff. All the defines from Hardwareconfig.h are visible since it is only included there.
The idea was to have the management of the config hidden from the user.
It is definitely not ideal, but works™ ;)
If you happen to have a better idea/solution I am all in for doing that

Reply:
I think it was just my python brain seeping in to my c brain and worrying about a problem that isnt there in c (that being circular imports). Moved it in to target.h and fixed my spelling error too :)